script/collect.js

```diff
@@ -181,22 +184,40 @@ const findVersionForTag = (releases, tag, source) => {
   throw new Error(`No release with tag '${tag}' found in ${source}!`)
 }
 
+/**
+ * Processes the release notes into HTML using hubdown.
+ * @param {*} release 
+ * @returns 
+ */
 async function processRelease(release) {
   release.version = release.tag_name.substring(1)
-  release.body = release.body
 
-    // turn PR references like #123 into hyperlinks
+  // turn PR references like #123 into hyperlinks
+  release.body = release.body
     .replace(
       / #(\d+)/gm,
       ' <a href="https://github.com/electron/electron/pull/$1">#$1</a>'
     )
 
-    // adjust heading levels (h2 becomes h3, etc)
```

Comment:
Technically unrelated change, but skipping heading levels is undesired. If this was for visual purposes in the website, we should just rather change the CSS.

https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks